workflow/rules/datavzrd.smk

```diff
@@ -0,0 +1,29 @@
+rule render_datavzrd_config_spia:
+    input:
+        template=workflow.source_path("../resources/datavzrd/spia-template.yaml"),
+        spia_results="results/tables/pathways/{model}.pathways.tsv",
+        barplot="results/plots/pathways/{model}.pathways.html",
+    output:
+        "results/datavzrd/spia/{model}.yaml"
+    log:
+        "logs/yte/render-datavzrd-config-spia/{model}.log"
+    template_engine:
+        "yte"
+
+rule datavzrd:
+    input:
+        config="results/datavzrd/{type}/{model}.yaml",
+        # files required for rendering the given configs
+        spia_results="results/tables/pathways/{model}.pathways.tsv",
+        barplot="results/plots/pathways/{model}.pathways.html",
+    output:
+        report(
+            directory("results/datavzrd-reports/{type}.{model}"),
+            htmlindex="index.html",
+            # see https://snakemake.readthedocs.io/en/stable/snakefiles/reporting.html
+            # for additional options like caption, categories and labels
+        ),
```

Comment:
Please add caption, category, etc. You can follow what has been done for the spia tsv output.

Reply:
Noted